test/data/psst-component-data/psst.json

```diff
@@ -1,12 +1,35 @@
 [
     {
+        "name": "a",
         "include": [
             "https://a.com/*"
         ],
         "exclude": [
         ],
         "version": 1,
-        "test_script": "a/test.js",
-        "policy_script": "a/policy.js"
+        "user_script": "user.js",
+        "policy_script": "policy.js"
+    },
+    {
+        "name": "b",
+        "include": [
+            "https://b.com/*"
+        ],
+        "exclude": [
+        ],
+        "version": 1,
+        "user_script": "user.js",
+        "policy_script": "policy.js"
+    },
+    {
+        "name": "rule_with_wrong_script_path",
+        "include": [
+            "https://url.com/*"
+        ],
+        "exclude": [
+        ],
+        "version": 1,
+        "user_script": "user.js",
```

Comment:
there should also be tests with only user_script or only policy_script

Reply:
for unit tests, we don't have to test this for browser tests

Reply:
>there should also be tests with only user_script or only policy_script 

agreed